frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.styled.tsx

```diff
@@ -1,5 +1,73 @@
 import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import Icon from "metabase/components/Icon";
+import Button from "metabase/core/components/Button";
 
-export const ExpressionWidgetWrapper = styled.div`
+export const Container = styled.div`
   width: 445px;
 `;
+
+export const FieldWrapper = styled.div`
+  padding: 0 1.5rem 1.5rem;
+`;
+
+export const ExpressionFieldWrapper = styled.div`
+  padding: 1.5rem 1.5rem 1rem;
+`;
+
+export const FieldTitle = styled.div`
+  margin-bottom: 0.5rem;
+
+  font-weight: 700;
+  font-size: 0.83em;
+  text-transform: uppercase;
+  letter-spacing: 0.06em;
+
+  color: ${color("text-light")};
+`;
+
+export const IconWrapper = styled.span`
+  margin-left: 4px;
+  cursor: help;
+
+  &:hover {
+    color: ${color("text-dark")};
+  }
+`;
+
+export const StyledFieldTitleIcon = styled(Icon)`
+  width: 10px;
+  height: 10px;
+`;
+
+export const Divider = styled.div`
+  width: 100%;
+  height: 1px;
+  background-color: #eeecec;
```

Comment:
We usually don't use custom colors because they cannot be changed in the admin UI. I think it should be something like `bg-light` or other `bg-` variants.